src/API/Baggage/Propagation/BaggagePropagator.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Baggage\Propagation;
+
+use OpenTelemetry\API\Baggage\Baggage;
+use OpenTelemetry\API\Baggage\BaggageBuilderInterface;
+use OpenTelemetry\API\Baggage\Entry;
```

Comment:
phan in CI seems to believe this isn't used.
https://github.com/open-telemetry/opentelemetry-php/pull/437/checks?check_run_id=3798402726#step:9:47